input_raw_test.go

```diff
@@ -30,22 +27,29 @@ func TestRAWInputIPv4(t *testing.T) {
 
 	listener, err := net.Listen("tcp", ":0")
 	if err != nil {
-		t.Fatal(err)
+		t.Error(err)
+		return
 	}
 	origin := &http.Server{
-		Handler:      http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {}),
+		Handler: http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			w.Write([]byte("ab"))
```

Comment:
Error check. Multiple places.

Reply:
We won't really use it anywhere! it will unnecessary allocation